app/controllers/posts_controller.rb

```diff
@@ -25,9 +25,19 @@ def edit
   # POST /posts
   def create
     @post = Post.new(post_params)
-
     if @post.save
-      redirect_to @post, notice: 'Post was successfully created.'
+      tag_params.split(",").each do |tag_new|
```

Comment:
W taki sposob jak robisz nie, musisz wiec znalezc inny sposob. Chodzi o to zeby napisac to tak, ze jak post bedzie ok, ale co z tagami bedzie nie ok (bo np dodamy validacje i nie bedzie spelniona), to zeby nie skonczylo sie tak, ze post masz zapisany w bazie, ale tagow nie, bo byl blad. Bo jak uzytkownik poprawi blad i wysle formualarz jeszcze raz, to znowu zapisze post (drugi raz). Itd.
Ale moze zostawmy to na koniec. Dodamy walidacje do tagow i bedziesz bardziej widzial o co mi chodzi.